### PR TITLE
pangomm: Fix broken pango includes

### PR DESCRIPTION
**Pull request by hexane360**
Changes includes in order to fix build errors caused by header
rearrangements in pango.

These are the pango changes which broken pangomm's build: https://gitlab.gnome.org/GNOME/pango/-/commit/d8df0da73752284dff28952224258b1884ed200c
This patch backports this commit in upstream pangomm: https://gitlab.gnome.org/GNOME/pangomm/-/commit/f065a2967e22658565f4228b73b1511d291e343f

Closes: https://trac.macports.org/ticket/64430

###### Type(s)

- [X] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.1 21C52 arm64
Xcode 13.2.1 13C100

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint --nitpick`?
- [N/A] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by ryandesign** [comment]
> This patch backports this commit in upstream pangomm: https://gitlab.gnome.org/GNOME/pangomm/-/commit/f065a2967e22658565f4228b73b1511d291e343f

Placing this URL at the top of the patchfile you're adding would make it easier, months down the road, to remember where the patch came from.

File: x11/pangomm/files/patch-pango-include.diff [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- pango/pangomm/attributes.h.orig	2022-01-25 14:42:31.000000000 -0500
```
Comment (ryandesign) [inline x11/pangomm/files/patch-pango-include.diff:1]:
```suggestion
https://gitlab.gnome.org/GNOME/pangomm/-/commit/f065a2967e22658565f4228b73b1511d291e343f
--- pango/pangomm/attributes.h.orig	2022-01-25 14:42:31.000000000 -0500
```

**Comment by hexane360** [comment]
Any updates on this? I've made the changes requested.

**Comment by pmetzger** [comment]
Merged.

**Pull request merged**